pkg/admission/annotations.go

```diff
@@ -0,0 +1,44 @@
+package admission
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	LastDownscaleAnnotationKey = "last-prepared-for-downscale"
```

Comment:
Can we prefix and name this similar to how the "no downscale" hook works? `grafana.com/prepared-for-downscale`?

Reply:
Done